web/gds-user-ui/src/components/ReviewSubmit/index.tsx

```diff
@@ -71,6 +72,11 @@ const ReviewSubmit: React.FC<ReviewSubmitProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [isSent]);
 
+  // Update the review step status to complete if the user submits a registration.
+  if (isTestNetSubmitted || isMainNetSubmitted) {
+    dispatch(setStepStatus({ step: 6, status: LSTATUS.COMPLETE }));
```

Comment:
What is the significance of the 6? Is it the step number?

Reply:
Yes, 6 is the review step number. There are constants for steps 1-5. I was unable to determine why 6 was excluded, since 6 is listed as a step number a few times in the code. However, I did notice some cases where the constants weren't used for other step numbers, so a story updating this should be added.